vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/permission_rules/per_day_limit_frequency_cap.cc

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/frequency_capping/permission_rules/per_day_limit_frequency_cap.h"
+#include "bat/ads/internal/frequency_capping/frequency_capping.h"
+#include "bat/ads/ads_client.h"
+#include "bat/ads/internal/time.h"
+#include "bat/ads/internal/client.h"
+
+#include "bat/ads/ad_info.h"
+
+namespace ads {
+
+bool PerDayLimitFrequencyCap::IsAllowed() {
+  auto respects_day_limit = AreAdsPerDayBelowAllowedThreshold();
+
+  std::ostringstream string_stream;
+  string_stream << "DoesHistoryRespectAdsPerDayLimit:    respects_day_limit: "
+      << respects_day_limit;
+  reason_for_exclusion_ = string_stream.str();
```

Comment:
Please change log message to "You have exceeded the allowed ads per day". We do not need the flag to be output. And only show message if `respect_day_limit` is `false`

Reply:
Done - good spot on the assignment as well - I'd overlooked it.